company-profiles/four-kitchens.md

```diff
@@ -2,10 +2,40 @@
 
 ## Company blurb
 
-⚠ We don't have much information about this company yet!
+[Four Kitchens](https://fourkitchens.com/) builds websites and apps for organizations that depend on large-scale or unconventional content.
 
-If you know something we don't, help us fill it in!  Here's how:
+We help our clients figure out what they need, build and migrate to new systems, and maintain the systems you have. We can be their full-service team, or we can augment your existing team.
 
-- Read our [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md)
-- Have a look at our [example company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md)
-- Follow the structure of the example profile and [send us a pull request with your changes to this file!](https://github.com/remoteintech/remote-jobs/edit/master/company-profiles/four-kitchens.md)
+We work with clients like NBCUniversal, NYU, and PRI.
+
+## Company size
+
+40 and growing. We are Engineers, UX Strategists, Project Managers, Support Engineers, Designers, Account Managers, and more.
+
+## Remote status
+
+In 2016, Four Kitchens ditched their office of ten years and became a fully-distributed company.
+
+We believe good talent is where you find it and, as a distributed team, we’re excited to be pioneers in what we believe is the future of work-culture. Even though we have no physical office, we strive to maintain a collaborative community using tools like Slack and Zoom. Our goal is for team members to feel like they are working together in the same room, even if they’re thousands of miles apart.
+
+Here's few articles we've written on remote/distributed culture:
+
+* [Going Distributed: First, Define Your Values](https://www.fourkitchens.com/blog/article/going-distributed-first-define-your-values/)
+* [Remote-First Thinking](https://www.fourkitchens.com/blog/article/step-two-remote-first-thinking/)
+* More articles in our [Distributed Focus series](https://www.fourkitchens.com/blog/series/distributed-focus/)
+
+## Region
+
+We hire full-time employees who live in the United States, and we hire contractors who live in North America or Latin America.
+
+## Company technologies
+
+A few of the technologies we work with include, but are not limited to: Drupal, WordPress, Javascript, Node, React, GraphQL, and building APIs.
+
+## Office locations
+
+None :tada:
```

Comment:
I'm going to go ahead and merge this, but you might want to consider updating this to use the emoji character :tada: instead of `:tada:` - there is nothing in place on the site that will convert this into an emoji.